### PR TITLE
Gpl123 json api for extraction lims merged release 12.1.0

### PR DESCRIPTION
**Pull request by emrojo**

### CODE REVIEW

File: app/controllers/api/v2/sample_metadata_controller.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module Api
+  module V2
+    # JSONAPI controller for {SamplesResource}
+    class SampleMetadataController < JSONAPI::ResourceController
```
Comment (JamesGlover) [inline app/controllers/api/v2/sample_metadata_controller.rb]: {SampleMetadataResource} not {SamplesResource}

Comment (emrojo) [reply app/controllers/api/v2/sample_metadata_controller.rb]: Thanks, I didnt see it

Comment (JamesGlover) [reply app/controllers/api/v2/sample_metadata_controller.rb]: Not a problem.
If you end up adding a new resource from scratch, there are also some scaffold that you might find useful: (Although not so useful here, where the resource already existed)

```bundle exec rails generate api_v2```

Might try and split it out into controllers and resources.

Comment (JamesGlover) [reply app/controllers/api/v2/sample_metadata_controller.rb]: Feel fre to merge this as soon as the tests pass.

**Pull request merged**